aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -0,0 +1,431 @@
+      "description": "A list of the DB security groups to assign to the DB instance. The list can include both the name of existing DB security groups or references to AWS::RDS::DBSecurityGroup resources created in the template."
+    },
+    "DBSnapshotIdentifier": {
+      "type": "string",
+      "description": "The name or Amazon Resource Name (ARN) of the DB snapshot that's used to restore the DB instance. If you're restoring from a shared manual DB snapshot, you must specify the ARN of the snapshot.",
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,254}$"
+    },
+    "DBSubnetGroupName": {
+      "type": "string",
+      "description": "A DB subnet group to associate with the DB instance. If you update this value, the new subnet group must be a subnet group in a new VPC.",
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,254}$"
+    },
+    "DeleteAutomatedBackups": {
+      "type": "boolean",
+      "description": "A value that indicates whether to remove automated backups immediately after the DB instance is deleted. This parameter isn't case-sensitive. The default is to remove automated backups immediately after the DB instance is deleted."
+    },
+    "DeletionProtection": {
+      "type": "boolean",
+      "description": "A value that indicates whether the DB instance has deletion protection enabled. The database can't be deleted when deletion protection is enabled. By default, deletion protection is disabled."
+    },
+    "Domain": {
+      "type": "string",
+      "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
+    },
+    "DomainIAMRoleName": {
+      "type": "string",
+      "description": "Specify the name of the IAM role to be used when making API calls to the Directory Service."
+    },
+    "EnableCloudwatchLogsExports": {
+      "type": "array",
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "items": {
+        "type": "string"
+      },
+      "description": "The list of log types that need to be enabled for exporting to CloudWatch Logs. The values in the list depend on the DB engine being used."
+    },
+    "EnableIAMDatabaseAuthentication": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable mapping of AWS Identity and Access Management (IAM) accounts to database accounts. By default, mapping is disabled."
+    },
+    "EnablePerformanceInsights": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable Performance Insights for the DB instance."
+    },
+    "Engine": {
+      "type": "string",
+      "enum": [
+        "aurora",
+        "aurora-mysql",
+        "aurora-postgresql",
+        "mariadb",
+        "mysql",
+        "oracle-ee",
+        "oracle-se2",
+        "oracle-se1",
+        "oracle-se",
+        "postgres",
+        "postgresql-license",
```

Comment:
"postgresql-license",

*aws-rds-dbinstance:198*

Could you elaborate this version name or provide a doc? 
I didn’t find it under engine valid values here : https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

Reply:
This came as a surprise to me when I was running contract tests.

If you run a basic create-db-instance for a postgres db:
```
aws rds --region us-east-1 create-db-instance --db-name mydb --db-instance-identifier mydbinstance --allocated-storage 20 --engine postgres --db-instance-class db.t3.micro --master-username=<username> --master-user-password=<password>
```
The output would contain:
```
"LicenseModel": "postgresql-license",
```